<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>924</width>
    <height>807</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pyCOVID</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>COVID-19.jpg</normaloff>COVID-19.jpg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>911</width>
      <height>261</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Search by LIMITATIONS</string>
     </attribute>
     <widget class="QPushButton" name="submitVALUE">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>120</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>251</width>
        <height>111</height>
       </rect>
      </property>
      <property name="title">
       <string>Date</string>
      </property>
      <widget class="QDateEdit" name="dateStart">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>40</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>1</month>
         <day>22</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEnd">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>70</y>
         <width>161</width>
         <height>22</height>
        </rect>
       </property>
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2020</year>
         <month>1</month>
         <day>22</day>
        </datetime>
       </property>
       <property name="displayFormat">
        <string>yyyy-MM-dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxDateStart">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxDateEnd">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>End</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>251</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Country/Region:</string>
      </property>
      <widget class="QComboBox" name="contySelect">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="editable">
        <bool>true</bool>
       </property>
       <property name="currentText">
        <string>*</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>331</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Values</string>
      </property>
      <widget class="QTabWidget" name="tabWidget_2">
       <property name="geometry">
        <rect>
         <x>6</x>
         <y>30</y>
         <width>321</width>
         <height>141</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>Confirmed</string>
        </attribute>
        <widget class="QLineEdit" name="CS">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="CE">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Recovered</string>
        </attribute>
        <widget class="QLineEdit" name="RS">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="RE">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>Deaths</string>
        </attribute>
        <widget class="QLineEdit" name="DS">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>10</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLineEdit" name="DE">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>40</y>
           <width>241</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>*</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>54</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>End</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>10</y>
        <width>191</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Columns</string>
      </property>
      <widget class="QCheckBox" name="checkBoxCR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>171</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Counrty/Region</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxDate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxC">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Confirmed</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>111</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Recovered</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBoxD">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Deaths</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="clear">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>90</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>COVID-19.jpg</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>SQL Intepreter</string>
     </attribute>
     <widget class="QPushButton" name="submitSQL">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SQLBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>801</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECT * FROM Country_CRD LIMIT 500;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_8">
     <attribute name="title">
      <string>History</string>
     </attribute>
     <widget class="QPlainTextEdit" name="History">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>10</y>
        <width>891</width>
        <height>211</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>725</y>
      <width>921</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>280</y>
      <width>921</width>
      <height>441</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>924</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>_</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About this Program</string>
   </property>
  </action>
  <action name="actionCSV">
   <property name="text">
    <string>Comma Separated Value (CSV)</string>
   </property>
  </action>
  <action name="actionTab_Separated_Value_TSV">
   <property name="text">
    <string>Tab Separated Value (TSV)</string>
   </property>
  </action>
  <action name="actionSQL_INSERT">
   <property name="text">
    <string>SQL INSERT</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export Result as...</string>
   </property>
  </action>
  <action name="actionAbout_COVID_19">
   <property name="text">
    <string>About COVID-19</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
